frontend/src/components/InlineMessage/InlineMessage.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { Meta, Story } from '@storybook/react'
+
+import { InlineMessage, InlineMessageProps } from './InlineMessage'
+
+export default {
+  title: 'Components/InlineMessage',
+  component: InlineMessage,
+} as Meta
+
+const InlineMessageTemplate: Story<InlineMessageProps> = (args) => (
+  <InlineMessage {...args} />
+)
+
+export const Default = InlineMessageTemplate.bind({})
+Default.args = {
+  variant: 'info',
+  children: `**Markdown** accepted, or just use a normal string.`,
+  useMarkdown: true,
```

Comment:
default markdown is false though, so maybe default can showcase the alternative usecase of rendering random things, then add a new `WithMarkdown` story that shows this current default story